app/assets/javascripts/questions.coffee

```diff
@@ -1,3 +1,11 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+$(document).ready ->
+$('.form-radio').click ->
```

Comment:
better name it like 'input.question.form-radio'
in this case we 100% sure that this is only for questions